xml/System.Net.Http/SocketsHttpConnectionContext.xml

```diff
@@ -55,8 +55,8 @@
         <ReturnType>System.Net.Http.HttpRequestMessage</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>The initial HttpRequestMessage that is causing the connection to be created.</summary>
+        <value>An HttpRequestMessage instance causing the connection to be created.</value>
```

Comment:
```suggestion
        <value>The request message that's causing the connection to be created.</value>
```

The actual type is shown automatically above the description. So we're not supposed to include it.